ecommerce/edunext/models.py

```diff
@@ -33,3 +34,8 @@ class Meta(object):
         Meta class for SiteOptions model
         """
         verbose_name_plural = "SiteOptions"
+
+    def save(self, *args, **kwargs):
+        # Clear Site cache upon SiteOptions changed
```

Comment:
If I'm not wrong, it lasts forever unless you clear it by code, or restart the app or change the site